README.md

```diff
@@ -0,0 +1,110 @@
+# Scalar Admin k8s
+
+## Usage of the CLI tool
+
+```console
+Usage: scalar-admin-k8s-cli [-h] [-d=<pauseDuration>] [-n=<namespace>]
+                            -r=<helmReleaseName> [-w=<maxPauseWaitTime>]
+                            [-z=<zoneId>]
+Scalar Admin pause tool for the Kubernetes environment
+  -d, --pause-duration=<pauseDuration>
+                             The duration of the pause period by millisecond.
+                               5000 (5 seconds) by default.
+  -h, --help                 Display the help message.
+  -n, --namespace=<namespace>
+                             Namespace that Scalar products you want to pause
+                               are deployed. `default` by default.
+  -r, --release-name=<helmReleaseName>
```

Comment:
To be clear, only `-r` is required, correct? If so, I think it would be helpful to note that this parameter is required in the parameter description here.